src/controller/users_controls.py

```diff
@@ -0,0 +1,19 @@
+from fastapi.encoders import jsonable_encoder
+from src.utils.functions import model
+from ..schema.users_model import user
+from ..utils.auth import hash
+    
+async def register(User):
+    try:
+        User=user()
+        mail= User.email
+        check= await model.findone(mail,"questions")
+        if check is not None:
+            return {"error: ": "user already exists"}
+        User.password = hash(User.password)
+        foo= await model.create(jasonable_encoder(User), "questions")
```

Comment:
Jasonable looks incorrect here, did you test?

Reply:
```suggestion
        foo= await model.create(jsonable_encoder(User), "questions")
```
why are we using foo? Please use a more descriptive name

Reply:
resolved